examples/js/loaders/GLTFLoader.js

```diff
@@ -270,19 +270,23 @@ THREE.GLTFLoader = ( function () {
 	 */
 	function GLTFLightsExtension( json ) {
 
-		this.name = EXTENSIONS.KHR_LIGHTS;
+		this.name = EXTENSIONS.KHR_LIGHTS_PUNCTUAL;
 
 		this.lights = {};
 
-		var extension = ( json.extensions && json.extensions[ EXTENSIONS.KHR_LIGHTS ] ) || {};
+		var extension = ( json.extensions && json.extensions[ EXTENSIONS.KHR_LIGHTS_PUNCTUAL ] ) || {};
 		var lights = extension.lights || {};
 
 		for ( var lightId in lights ) {
 
 			var light = lights[ lightId ];
 			var lightNode;
 
-			var color = new THREE.Color().fromArray( light.color );
+			// the color default value is [1, 1, 1]
+			var color = new THREE.Color(1, 1, 1);
```

Comment:
nit: the three.js docs recommend using hex for the constructor, so let's do `new THREE.Color( 0xffffff )` here.